models/prices/prices_tokens.sql

```diff
@@ -279,6 +279,7 @@ VALUES
     ("amn-amon", "ethereum", "AMN", "0x737f98ac8ca59f2c68ad658e3c3d8c8963e40a4c", 18),
     ("amp-amp", "ethereum", "AMP", "0xff20817765cb7f73d4bde2e66e067e58d11095c2", 18),
     ("ampl-ampleforth", "ethereum", "AMPL", "0xd46ba6d942050d489dbd938a2c909a5d5039a161", 9),
+    ("angle-angle", "ethereum", "ANGLE", "0x31429d1856ad1377a8a0079410b297e1a9e214c2", 18),
```

Comment:
we use coinpaprika for pulling prices & this id appears invalid: https://api.coinpaprika.com/v1/coins/angle-angle